lib/executable.dart

```diff
@@ -103,16 +102,6 @@ Future<void> main(List<String> args) async {
         artifacts: globals.artifacts,
       ),
       ConfigCommand(verboseHelp: verboseHelp),
-      CustomDevicesCommand(
-        customDevicesConfig: globals.customDevicesConfig,
-        operatingSystemUtils: globals.os,
-        terminal: globals.terminal,
-        platform: globals.platform,
-        featureFlags: featureFlags,
-        processManager: globals.processManager,
-        fileSystem: globals.fs,
-        logger: globals.logger,
-      ),
       DaemonCommand(hidden: !verboseHelp),
```

Comment:
Can we also remove `DaemonCommand` if it's not supported?

Reply:
If I remember correctly, the daemon command worked without issue when I tested before. The command is hidden by default so it's not listed in the document.

Reply:
Ah I see, I asked because it was listed as an unsupported command.